src/sentry/services/hybrid_cloud/organization_mapping/impl.py

```diff
@@ -48,14 +49,36 @@ def create(
 
         return serialize_organization_mapping(org_mapping)
 
-    def update(self, organization_id: int, update: RpcOrganizationMappingUpdate) -> None:
+    def reserve_slug_for_organization(
+        self,
+        *,
+        organization_id: int,
+        slug: str,
+        name: str,
+        region_name: str,
+        idempotency_key: Optional[str] = "",
+        customer_id: Optional[str] = None,
+        status: Optional[OrganizationStatus] = None,
```

Comment:
Should this parameter be used?

Reply:
😅 Missed this when removing it form the definition